modules/licensify/manifests/apps/licensify.pp

```diff
@@ -16,7 +16,7 @@
     require                        => File['/etc/licensing'],
     proxy_http_version_1_1_enabled => true,
     log_format_is_json             => true,
-    collectd_process_regex         => 'java -Duser.dir=/data/vhost/licensify.integration.publishing.service.gov.uk/licensify-.*',
+    collectd_process_regex         => 'java -Duser.dir=\/data\/vhost\/licensify\..*publishing\.service\.gov\.uk\/licensify-.*',
```

Comment:
Hmmm will this fail on Production because we don't have `production.publishing.service.gov.uk`? Think it needs a dot removed...?

```suggestion
    collectd_process_regex         => 'java -Duser.dir=\/data\/vhost\/licensify\.*publishing\.service\.gov\.uk\/licensify-.*',
```

Reply:
Whoops, misread this - you're only escaping one `\.`, not two - so as you've said IRL, this is fine 😁